packages/radio-player/src/assets/img/left.ts

```diff
@@ -0,0 +1,17 @@
+import { html } from 'lit-html';
+
+const image = html`
```

Comment:
question: what is the difference between `left.svg` and `left.ts`?

Reply:
I didn't see the `ts` versions used, perhaps delete them?

Reply:
Ah actually we use the `left.ts` one because it's kind of a pain to distribute `svg` files with npm. I'll delete the `svg` files.

Reply:
Done.